database/clickhouse/clickhouse_test.go

```diff
@@ -106,6 +106,33 @@ func testSimple(t *testing.T, engine string) {
 	})
 }
 
+func testSimpleWithInstance(t *testing.T) {
```

Comment:
Add a comment that this is intended to test setting default config values with instances (or rename the test accordingly)

Reply:
renamed to `testSimpleWithInstanceDefaultConfigValues`.